comments-bundle/contao/dca/tl_comments.php

```diff
@@ -493,12 +493,14 @@ public function listComments($arrRow)
 	{
 		$router = System::getContainer()->get('router');
 		$title = $GLOBALS['TL_LANG']['tl_comments'][$arrRow['source']] . ' ' . $arrRow['parent'];
+		$db = Database::getInstance();
```

Comment:
Same here.

Reply:
Changed in 21fdc957ab098dba352f984b67bb8c4546d3eb05.